examples/rllib/multiagent_exps/multiagent_traffic_light_grid.py

```diff
@@ -0,0 +1,282 @@
+                "discrete": False,
+                "tl_type": "actuated",
+                "num_local_edges": 4,
+                "num_local_lights": 4,
+            },
+        ),
+
+        # network-related parameters (see flow.core.params.NetParams and the
+        # scenario's documentation or ADDITIONAL_NET_PARAMS component)
+        net=NetParams(
+            inflows=inflow,
+            no_internal_links=False,
+            additional_params={
+                "speed_limit": V_ENTER + 5,
+                "grid_array": {
+                    "short_length": SHORT_LENGTH,
+                    "inner_length": INNER_LENGTH,
+                    "long_length": LONG_LENGTH,
+                    "row_num": n_rows,
+                    "col_num": n_columns,
+                    "cars_left": N_LEFT,
+                    "cars_right": N_RIGHT,
+                    "cars_top": N_TOP,
+                    "cars_bot": N_BOTTOM,
+                },
+                "horizontal_lanes": 1,
+                "vertical_lanes": 1,
+            },
+        ),
+
+        # vehicles to be placed in the network at the start of a rollout (see
+        # flow.core.params.VehicleParams)
+        veh=vehicles,
+
+        # parameters specifying the positioning of vehicles upon initialization
+        # or reset (see flow.core.params.InitialConfig)
+        initial=InitialConfig(
+            spacing='custom',
+            shuffle=True,
+        ),
+    )
+    return flow_params
+
+
+def setup_exps_PPO(flow_params):
+    """
+    Experiment setup with PPO using RLlib.
+
+    Returns
+    -------
+    str
+        name of the training algorithm
+    str
+        name of the gym environment to be trained
+    dict
+        training configuration parameters
+    """
+    alg_run = 'PPO'
+    agent_cls = get_agent_class(alg_run)
+    config = agent_cls._default_config.copy()
+    config["num_workers"] = min(N_CPUS, N_ROLLOUTS)
+    config['train_batch_size'] = HORIZON * N_ROLLOUTS
+    config['simple_optimizer'] = True
+    config['gamma'] = 0.999  # discount rate
+    config['model'].update({'fcnet_hiddens': [32, 32]})
+    config['lr'] = tune.grid_search([1e-5, 1e-4, 1e-3])
+    config['horizon'] = HORIZON
+    config['clip_actions'] = False  # FIXME(ev) temporary ray bug
+    config['observation_filter'] = 'NoFilter'
+
+    # save the flow params for replay
+    flow_json = json.dumps(
+        flow_params, cls=FlowParamsEncoder, sort_keys=True, indent=4)
+    config['env_config']['flow_params'] = flow_json
+    config['env_config']['run'] = alg_run
+
+    create_env, env_name = make_create_env(params=flow_params, version=0)
+
+    # Register as rllib env
+    register_env(env_name, create_env)
+
+    test_env = create_env()
+    obs_space = test_env.observation_space
+    act_space = test_env.action_space
+
+    def gen_policy():
+        return (PPOPolicyGraph, obs_space, act_space, {})
+
+    # Setup PG with an ensemble of `num_policies` different policy graphs
```

Comment:
This comment is slightly misleading, since you're actually just setting it up with 1 policy graph.